server/odc-service/src/main/java/com/oceanbase/odc/service/session/util/DBSchemaExtractor.java

```diff
@@ -200,6 +200,36 @@ private static class OBMySQLRelationFactorVisitor extends OBParserBaseVisitor<Re
 
         private final Set<DBSchemaIdentity> identities = new HashSet<>();
 
+        public RelationFactor visitDrop_function_stmt(
```

Comment:
this may be a common problem not just related to pl edition, please submit a issue first.

you may forget to process ob-oracle mode

Reply:
For the editing pl of ob mysql, this function includes drop procedure. This pr can only solve the drop statement of ob mysql to ensure the normal operation of pl editing function. 
executing drop procedure statement in sql windows does not fall under the scope of pl editing and should fall under the scope of sql window